api/models/report.py

```diff
@@ -19,7 +19,7 @@ class Report(db.Document):
     downpayment_percentage = db.IntField(required=True)
     goal_principal = db.IntField(default=0)
     rent = db.IntField(default=0)
-    uid = db.StringField(unique=True)
+    uid = db.StringField()
```

Comment:
Debugging solve:
- First we tested POST for /report in localhost through Postman
- This did not present the same break as our production server
- We then pulled up `heroku logs --tail`
- This lead us to a duplicate key error (because empty uid's were reporting as null, conflicts occurred)

**We are removing until we can discover a method for forcing unique but ignoring null**